besu/src/test/java/org/hyperledger/besu/cli/CommandLineUtilsTest.java

```diff
@@ -53,9 +64,19 @@ private abstract static class AbstractTestCommand implements Runnable {
 
     final CommandLine commandLine;
 
-    AbstractTestCommand(final Logger logger) {
+    final Map<String, String> environment = new HashMap<>();
+
+    AbstractTestCommand(final Logger logger, final Optional<File> configFile) {
       this.logger = logger;
       commandLine = new CommandLine(this);
+      if (configFile.isPresent()) {
+        commandLine.setDefaultValueProvider(
```

Comment:
Rather than setting the provider here and having to pass the configFile up the class hierarchy, I think it would be better to set it directly in the `multipleMainOptionsToml`, especially since that's the only test than needs it.

You should have access to everything you need via `testCommand`